app/src/main/java/org/fossasia/pslab/communication/ScienceLab.java

```diff
@@ -1075,7 +1075,7 @@ private int calcCHOSA(String channelName) {
     public double getVoltage(String channelName, Integer sample) {
         this.voltmeterAutoRange(channelName);
         double Voltage = this.getAverageVoltage(channelName, sample);
-        if (channelName.equals("CH2") || channelName.equals("CH1") ) {
+        if (channelName.equals("CH2") || channelName.equals("CH1")) {
```

Comment:
It's better to use string literal first. "CH2".equals(channelName)

Reply:
This specific change is due to refractor code.